### DISCUSSION TITLE
Zoom function

### DISCUSSION THREAD

**Discussion by Charles-770** [sumatrapdfreader/sumatrapdf#3524] [Ideas]
Thanks for your wonderful work!

Please, could you add in the Zoom function, a command to return to 100%, 
as well as the display of the current magnification.

**Comment by GitHubRulesOK** [comment]
Ctrl + Y should show current, here it is in Acrobat (currently comparing that on another device Note 100% is also another name for 1:1) but its very similar in SumatraPDF
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/188e99f3-c46c-470d-aaf3-881a6e3bc515)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/ddc7720d-b18d-46e2-988f-92461f7016e1)

For keyboard its 
<html>
<body>


CmdZoomActualSize | Ctrl + 1 | Zoom: Actual Size
-- | -- | --



</body>
</html>

So you can use Ctrl + 1 or set your own say Z

**Reply by Charles-770** [reply]
Thanks, but we should have it displayed in the bar (or at least be able to choose to display it).

**Reply by GitHubRulesOK** [reply]
Related already raised issues
https://github.com/sumatrapdfreader/sumatrapdf/issues/2237 https://github.com/sumatrapdfreader/sumatrapdf/issues/1117
canny requests https://sumatrapdf.canny.io/feature-requests/p/toolbar-customizability  
The problem with putting every command on the page like Adobe do is its a lot of clutter for commands not often used so just a hotkey such as Ctrl1 avoids that hence SumatraPDF is light and fast
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/7a5d4c30-1907-4eeb-bd8c-8ec1e47ec4d5)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/2199533f-5235-4d8c-85ec-712ddb4a2cd5)
